backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/ResourceManager.java

```diff
@@ -311,6 +312,9 @@ public void internalSetVmStatus(VmDynamic vm,
                     vm.setCurrentCoresPerSocket(0);
                     vm.setCurrentThreadsPerCore(0);
                     vm.setCurrentNumaPinning(null);
+                    if (getVmManager(vm.getId()).getCpuPinningPolicy() == CpuPinningPolicy.RESIZE_AND_PIN_NUMA) {
```

Comment:
this part breaks the basic design of vms monitoring - the way to do this is by detecting that we are changing a vm with the resize-and-pin policy to Down state in VmAnalyzer and save it there so when VmsMonitoring gets to postFlush it can check this and trigger the flow that would end up with VdsEventListener#clearVmNuma

Reply:
well, it changes the database so probably not in postFlush but on flush